From 64a064b67b6730fee0ce1871897f8cddbdd6c1bb Mon Sep 17 00:00:00 2001
From: Elian Doran <contact@eliandoran.me>
Date: Sat, 30 Dec 2023 19:39:45 +0200
Subject: [PATCH] Add global pro

---
 sqlitestudio.pro | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 sqlitestudio.pro

diff --git a/sqlitestudio.pro b/sqlitestudio.pro
new file mode 100644
index 00000000..89052a61
--- /dev/null
+++ b/sqlitestudio.pro
@@ -0,0 +1,2 @@
+TEMPLATE = subdirs
+SUBDIRS = SQLiteStudio3 Plugins

Plugins.depends = SQLiteStudio3
-- 
2.42.0

