From 18091b91c74d4c6b27f5d3045780982a07acee18 Mon Sep 17 00:00:00 2001
From: Elian Doran <contact@eliandoran.me>
Date: Sun, 21 Apr 2024 18:33:31 +0300
Subject: [PATCH] Downgrade kf

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2e2821c..a7e36cc 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -11,7 +11,7 @@ project(marknote VERSION ${RELEASE_SERVICE_VERSION})
 include(FeatureSummary)
 
 set(QT_MIN_VERSION 6.4)
-set(KF_MIN_VERSION 5.240)
+set(KF_MIN_VERSION 5.115)
 
 find_package(ECM ${KF_MIN_VERSION} REQUIRED NO_MODULE)
 
-- 
2.42.0

